src/middlewares/validator/validator.js

```diff
@@ -23,7 +23,8 @@ const body = joi.string().required().error(() => 'Body is required');
 const tagList = joi.array().required().error(() => 'tagList must be an array ');
 const plainText = joi.string().required().error(() => 'Plain text is required');
 
-const articleId = joi.string().guid().required().error(() => 'We do not know the article you are trying to rate. Please provide a valid article id');
+const articleId = joi.string().guid().required().error(() => 'We do not know the article you are referring to rate. Please provide a valid article id');
+const commentId = joi.string().guid().required().error(() => 'We do not know the comment you are referring to. Please provide a comment id');
```

Comment:
You have done a nice job with the way you implemented validation here but the 3 lines of code above are long. You may need to make them shorter to improve the readability.

Reply:
Thanks for the review Kayroy247

Readability is an important part of coding.
I would implement the suggested changes now

Thanks again